features/image-transformations/overlay-using-layers.md

```diff
@@ -0,0 +1,220 @@
+| [cm](../image-transformations/resize-crop-and-other-transformations#crop-crop-modes-and-focus)              | Crop mode. Supports `extract` and `pad_resize`. |
+| [fo](../image-transformations/resize-crop-and-other-transformations#focus-fo)                               | Relative focus area used during cropping. Accepts `center`, `top`, `left`, `bottom`, `right`, `top_left`, `top_right`, `bottom_left`, and `bottom_right`. |
+| [b](../image-transformations/resize-crop-and-other-transformations#border-b)                                | This adds a border to the overlay image. It accepts two parameters - the width of the border and the color of the border in format `b-<border-width>-<hex code>` |
+| [bg](../image-transformations/resize-crop-and-other-transformations#background-color-bg)                    | It is used to specify the background color in RGB Hex Code (e.g. FF0000) or an RGBA Code (e.g. FFAABB50) that must be used for the image. If you specify an 8 character background, the last two characters must be a number between 00 and 99, which is used to indicate the opacity level of the background. `00` represents an opacity level of `0.00`, `01` represents an opacity level of `0.01`, and so on. |
+| [r](../image-transformations/resize-crop-and-other-transformations#radius-r)                                | It is used to control the radius of the corner. To get a circle or oval shape, set the value to `max`. |
+| [rt](../image-transformations/resize-crop-and-other-transformations#rotate-rt)                              | It is used to specify the degree by which the overlay image must be rotated. |
+| [t](../image-transformations/resize-crop-and-other-transformations#trim-edges-t)                            | By default, ImageKit.io trims the overlay image before overlaying it on the base image. Trimming removes similar colored pixels from the edges. To prevent the default trimming from happening, use `t-false`. **Possible values** include `true`, `false`, and integer values between `1` and `99` that specifies the threshold level for considering a particular pixel as "background". |
+| [dpr](../image-transformations/resize-crop-and-other-transformations#dpr-dpr)                               | It is used to specify the device pixel ratio that is used to calculate the dimensions of the overlay image. It can only be used when either the height or width of the desired output image is specified. The possible values are `0.1` to `5`, or `auto`. |
+| [q](../image-transformations/resize-crop-and-other-transformations#quality-q)                               | It is used to specify the quality of the output image for lossy formats like JPEG, WebP, and AVIF. A large quality number indicates a larger output image size with high quality. A small quality number indicates a smaller output image size with lower quality. It can be a whole number between `1` and `100`. The default value is `80`. |
```

Comment:
Following properties are missing that should work for images but don't for video at the moment:
* blur
* https://docs.imagekit.io/features/image-transformations/resize-crop-and-other-transformations#grayscale-e-grayscale
* https://docs.imagekit.io/features/image-transformations/image-enhancement-and-color-manipulation#contrast-stretch-e-contrast
* https://docs.imagekit.io/features/image-transformations/image-enhancement-and-color-manipulation#sharpen-e-sharpen
* https://docs.imagekit.io/features/image-transformations/image-enhancement-and-color-manipulation#unsharp-mask-e-usm

Reply:
added